wordpress-mobile/WordPress-Android
# Small stats design improvements

Fixes #9305 #9306 #9303 #9292 
This PR fixes several small details in stats. 
* I'm changing the image type of icons to `ICON` and of the avatar to `AVATAR_WITH_BACKGROUND` as they show the proper placeholders.
* In referrers I'm replacing the icon in `search engines` with a gridicon
* In referrers there is now a default `globe` icon if the icon is missing
* I'm removing the default loading empty message from the bar chart
* The chevron's size in the expandable item is now 16dp and the color is changed to `grey_darken_20` 

To test:
* Open Stats/DWMY
* Check all the above points
